### PR TITLE
iob: enhance fuzzer with bits for SLEW and DRIVE for LVCMOS25

### PR DESCRIPTION
**Pull request by tmichalak**

### CODE REVIEW

File: fuzzers/030-iob/generate.py [hunk 1]
```diff
@@ -7,27 +7,42 @@
 
 print("Loading tags")
 '''
-port,site,tile,pin,val
+port,site,tile,pin,slew,drive,pulltype
```
Comment (litghost) [inline fuzzers/030-iob/generate.py]: Should have both iostandard and drive?

File: fuzzers/030-iob/generate.py [hunk 2]
```diff
@@ -7,27 +7,42 @@
...
-        val = "NONE"
+    if pulltype == "":
+        pulltype = "NONE"
     segmaker.add_site_group_zero(
         segmk, site, "PULLTYPE.", ("NONE", "KEEPER", "PULLDOWN", "PULLUP"),
-        "PULLDOWN", val)
+        "PULLDOWN", pulltype)
 
+    segmaker.add_site_group_zero(
+        segmk, site, "LVCMOS25.DRIVE.", ("4", "8", "12", "16"), "12", drive)
```
Comment (litghost) [inline fuzzers/030-iob/generate.py]: Use the IOSTANDARD from the CSV?

File: fuzzers/030-iob/generate.tcl [hunk 3]
```diff
@@ -53,20 +53,26 @@ proc loc_pins {} {
         set tile [get_tiles -of_objects $site]
         set pin [dict get $io_pin_sites $site]
 
-        set_property -dict "PACKAGE_PIN $pin IOSTANDARD LVCMOS33" $port
+        set_property -dict "PACKAGE_PIN $pin IOSTANDARD LVCMOS25" $port
```
Comment (litghost) [inline fuzzers/030-iob/generate.tcl]: Toggle between 2.5 and 3.3?

File: fuzzers/030-iob/bits.dbf [hunk 4]
```diff
@@ -1 +1,3 @@
 38_98 39_97 39_99,LIOB33.IOB_Y1.PULLTYPE.PULLDOWN
```
Comment (litghost) [inline fuzzers/030-iob/bits.dbf]: What happened to IOB_Y0?

Comment (tmichalak) [reply fuzzers/030-iob/bits.dbf]: I think IOB_Y1 is used for inputs whereas IOB_Y0 for outputs. Drive and slew are properties for outputs only. Do you think we should have a separate entry in the dbf file for IOB_Y0 that is exactly as for IOB_Y1?

File: fuzzers/030-iob/generate.tcl (hunk 3 above)
Comment (tmichalak) [reply fuzzers/030-iob/generate.tcl]: Well, the thing is that for LVCMOS33 we will solve the pulltype and slew bits, but not the drive - no clear zero value group problem.

File: fuzzers/030-iob/bits.dbf (hunk 4 above)
Comment (litghost) [reply fuzzers/030-iob/bits.dbf]: That is incorrect.  IOB_Y1 and IOB_Y0 can be both inputs and outputs.

File: fuzzers/030-iob/generate.tcl (hunk 3 above)
Comment (litghost) [reply fuzzers/030-iob/generate.tcl]: A zero value group is not required if it doesn't exist.

**Comment by litghost** [comment]
Please rebase and re-test this PR on master.

File: fuzzers/030-iob/bits.dbf (hunk 4 above)
Comment (tmichalak) [reply fuzzers/030-iob/bits.dbf]: I state corrected, I rebased the PR and now we have IOB_Y0 as well as IOB_Y1

**Pull request merged**